docs/integrations/connector-support-levels.md

```diff
@@ -64,5 +64,10 @@ necessary to ensure that the Connector Catalog maintains a minimum level of qual
 Archived connectors will not receive any further updates or support from the Airbyte team. Archived
 connectors remain source-available in the
 [`airbytehq/connector-archive`](https://github.com/airbytehq/connector-archive) repository on
-GitHub. If you wish to take over the maintenance of an archived connector, please open a Github
-Discussion.
+GitHub.
+
+If you wish to take over the maintenance of an archived connector, please open a Github Discussion.
+For API Sources (python), updating the connector to the latest version of the
+[CDK](/connector-development/cdk-python/) and ensuring that the connector successfully passes the
+[Connector Acceptance Tests](/connector-development/testing-connectors/connector-acceptance-tests-reference)
+is the start to the un-archiving process.
```

Comment:
Airbyte requires the acceptance test to function in the CI pipeline. This requires access to a sandbox account for proper testing. Some service providers may not allow or support this, making it challenging for the Airbyte project to maintain the connector.

@evantahler wdyt about adding this notes? Some sources we don't have sandbox and they're very hard to get them: Genesys, Ashby and others.

Reply:
This is a page about connector archival.  I think that note might belong in a more general doc about what connectors we are willing to accept, but not here.

Reply:
Makes sense I'll work to update the **Connector Acceptance Tests** to reflect that.